config.json

```diff
@@ -3,6 +3,72 @@
   "language": "Elixir",
   "repository": "https://github.com/exercism/xelixir",
   "active": true,
+  "problems": [
```

Comment:
Hey @ananthamapod the "problems" key seems to have snuck back into the config.json with the PR. Please remove.